tree/ntuple/v7/inc/ROOT/RPageStorage.hxx

```diff
@@ -379,7 +379,7 @@ protected:
    public:
       void Insert(DescriptorId_t physicalColumnID);
       void Erase(DescriptorId_t physicalColumnID);
-      RCluster::ColumnSet_t ToColumnSet();
+      RCluster::ColumnSet_t ToColumnSet() const;
```

Comment:
As a side note: I also thought we could keep a single `std::unordered_map<DescriptorId_t, std::size_t>` that maps a column ID to its ref count (in substitution of both vectors).  Then, `ToColumnSet()` might construct a `unordered_set` from the map, which might be faster (not sure about that) as we can, e.g. initialize it with the proper number of buckets.

Not a requirement for this PR, I think, though, but we may consider it in the future.